primefaces/src/main/java/org/primefaces/virusscan/impl/ClamDaemonClient.java

```diff
@@ -48,34 +48,56 @@ public class ClamDaemonClient {
 
     // "do not exceed StreamMaxLength as defined in clamd.conf, otherwise clamd
     // will reply with INSTREAM size limit exceeded and close the connection."
-    private static final int CHUNK_SIZE = 2048;
-    private static final int DEFAULT_TIMEOUT = 30000;
+    public static final int DEFAULT_BUFFER = 2048;
+    public static final String DEFAULT_HOST = "localhost";
+    public static final int DEFAULT_PORT = 3310;
+    public static final int DEFAULT_TIMEOUT = 30000;
 
-    private final String hostName;
+    private final int buffer;
```

Comment:
Why a rename of chunkSize and hostName? This would affect people that have extended this class.

Reply:
this was not possible yet. since these attribute we private uptonow.

Reply:
the test are not running through. they should not be affected by my commits.

Reply:
Ah yes, there were no getters.